environment

```diff
@@ -73,6 +73,9 @@ DSS_GS_CHECKOUT_BUCKET_TEST_USER=org-hca-dss-checkout-test-user  # for tests w/n
 DSS_GS_CHECKOUT_BUCKET_INTEGRATION=org-humancellatlas-dss-checkout-integration
 DSS_GS_CHECKOUT_BUCKET_STAGING=org-humancellatlas-dss-checkout-staging
 DSS_GS_CHECKOUT_BUCKET_PROD=org-humancellatlas-dss-checkout-prod
+DSS_INFRA_TAG_PROJECT=dcp
+DSS_INFRA_TAG_SERVICE=dss
+DSS_INFRA_TAG_ENV=${DSS_DEPLOYMENT_STAGE}
```

Comment:
This is already available to `/infra` scripting as `var.DSS_DEPLOYMENT_STAGE`

Reply:
changed in `local.common.tag` + removed